deploy/charts/alluxio/values.yaml

```diff
@@ -332,6 +332,29 @@ csi:
   # whether to launch CSI components
   enabled: false
 
+
+## ETCD cluster ##
+# For coordination operation for alluxio components such as
+# worker membership management
+etcd:
+  # by default etcd is not launched as part of Alluxio deployment
```

Comment:
If etcd is not launched by default, we shouldn't set `alluxio.worker.membership.type=ETCD` by default right?

Reply:
good catch let me fix with a condition here